app/controllers/mailing_list/steps_controller.rb

```diff
@@ -3,6 +3,8 @@ class StepsController < ApplicationController
     include WizardSteps
     self.wizard_class = MailingList::Wizard
 
+    before_action :set_page_title
```

Comment:
I don't think this action will work on pages which don't have the current step variable set so it should be excluding some of the actions - see

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/controllers/concerns/wizard_steps.rb#L6